analytics-custom-data-provider/pom.xml

```diff
@@ -105,10 +111,8 @@
         <maven.compiler.source>8</maven.compiler.source>
         <maven.compiler.target>8</maven.compiler.target>
 <!--        Below versions can be found by searching the component name in <APIM_HOME>/repository/components/plugins directory.-->
-<!--        org.wso2.carbon.apimgt.gateway component version-->
-        <carbon.apimgt.gateway.version>9.0.174.182</carbon.apimgt.gateway.version>
-<!--        org.wso2.carbon.apimgt.common.analytics component version-->
-        <carbon.apimgt.common.analytics.version>9.0.174.182</carbon.apimgt.common.analytics.version>
+<!--        org.wso2.carbon.apimgt component version-->
+        <carbon.apimgt.version>9.0.174.197</carbon.apimgt.version>
```

Comment:
This version can change, right? Moving forward, this pack will contain higher minor component levels (greater than 197) each time the customer update to the latest U2 level. Hence, we might need to say something like, 9.0.174.<current_minor_component_level> for future customers, shouldn't we?

Reply:
+1 and fixed with https://github.com/wso2/samples-apim/pull/105/commits/db2e763493f7503269ea30dbb2bccb1b97612d43

thanks!